internal/storage/metricstore/elasticsearch/processor.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package elasticsearch
+
+import (
+	"math"
+
+	"github.com/jaegertracing/jaeger/internal/proto-gen/api_v2/metrics"
+	"github.com/jaegertracing/jaeger/internal/storage/v1/api/metricstore"
+)
+
+// ProcessLatencies processes raw latency metrics by applying scaling and rounding.
+func ProcessLatencies(mf *metrics.MetricFamily) *metrics.MetricFamily {
```

Comment:
please think of a better naming. Saying "process" in this context is almost equivalent to "do something with it". Methods like `process()` are good if we actually have swappable components like the processors in OTEL pipeline - the pipeline doesn't know what "process" mean in each implementation, so "do something" connotation is actually reasonable. But here it's way too generic, as you don't have generic "Processor" components, you have static functions called explicitly by the business logic with precise expectation of that they are supposed to do. So the method names should reflect what they are doing, and not be called "process()"